modules/metastore/modules/metastore_admin/metastore_admin.install

```diff
@@ -25,3 +25,10 @@ function metastore_admin_uninstall() {
   $config->save(TRUE);
   drupal_flush_all_caches();
 }
+
```

Comment:
@dharizza I think this needs to enable the new module as well? I tried the upgrade QA path and got:
```
[error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
Configuration <em class="placeholder">core.entity_form_display.node.data.default</em> depends on the <em class="placeholder">JSON Form Widget</em> module that will not be installed after import. in Drupal\Core\Config\ConfigImporter->validate() (line 748 of /var/www/docroot/core/lib/Drupal/Core/Config/ConfigImporter.php). 
```

Reply:
Good catch, just fixed it. Thank you!